_includes/components/header--basic.html

```diff
@@ -50,12 +50,9 @@
         {% endif %}
 
         {% assign _secondary = header.secondary %}
-          {% if site.search  %}
+          {% if site.search_site_handle  %}
             <form accept-charset="UTF-8" action="https://search.usa.gov/search" id="search_form" method="get" class="usa-search usa-search-small js-search-form">
-              <div style="margin:0;padding:0;display:inline">
-                <input name="utf8" type="hidden" value="&#x2713;" />
```

Comment:
I would leave the hidden `<input>` tag in there since that maybe doing something important.